### ISSUE TITLE
Add flag to force ghidra to use a specific java jvm even if it is not officially supported.

### ISSUE THREAD

**Issue by Deflaktor** [NationalSecurityAgency/ghidra#1911]
**Is your feature request related to a problem? Please describe.**
I want to run ghidra server on my odroid which runs on ARM architecture. I am able to set up the server and get it running. However, when trying to add a user using the svrAdmin script it fails. It says that it needs a java version running on 64bit architecture. 

**Describe the solution you'd like**
A flag or setting to force ghidra to use a specifc java version that I provide even if it is unsupported. I take all risk in that decision. There is a JAVA_HOME_OVERRIDE setting, however ghidra still checks if it supports this particular java version and just exits with error code 1 if it does not.

**Describe alternatives you've considered**
Modifying the source code to disable that check.

![grafik](https://user-images.githubusercontent.com/4169014/83353319-0b45f000-a352-11ea-9009-8d4562fd8f19.png)

**Comment by asgmeonerandom** [comment]
where is the problem exactly?
It asks you to enter the path for JDK. There must be a reason that only 64 bit is supported...

**Comment by Deflaktor** [comment]
Well the odroid runs on ARM architecture. There is only the 32 bit ARM jvm available so it is not possible to get a 64 bit jvm on here. 
The reason why only 64 bit is supported is because of #882 which is completely unrelated to the server administration feature. So it is an artifical precondition which in this specific case is not really needed.

**Comment by asgmeonerandom** [comment]
> Well the odroid runs on ARM architecture. There is only the 32 bit ARM jvm available so it is not possible to get a 64 bit jvm on here.
> The reason why only 64 bit is supported is because of #882 which is completely unrelated to the server administration feature. So it is an artifical precondition which in this specific case is not really needed.

`NOTE: All 32-bit OS installations are now deprecated. Please contact the Ghidra team if you have a specific need.`  from https://ghidra-sre.org/InstallationGuide.html

**Comment by ryanmkurtz** [comment]
Because this is such a specialized use case, I recommend you just go into the script where you need the check disabled (in this case launch.sh) and comment out the check and hardcode in the JAVA_HOME you want to use.  Note that launch.sh is shared between svrAdmin and ghidraRun so you may want to make the changes in a copy of launch.sh and have svrAdmin call that.

**Comment by Deflaktor** [comment]
Thanks @ryanmkurtz, I was able to work it out by modifying the launch.sh. I missed that this is a possibility.
I created this issue under the assumption that I need to clone the whole repo undo commit 0a3574d0c07887f105854747d6153614dcfe4be4 and then compile LaunchSupport.jar to get this working. If it is just a simple script change then I guess there is no need for such a feature. 

For anyone wondering how to setup the ghidra server on a 32bit system, here are the steps:
https://gist.github.com/Deflaktor/771986f94507751f850d25d13fa8385e
